net-dns/dnsviz/dnsviz-0.8.2.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+inherit distutils-r1
+
+MY_PN="dnsviz"
```

Comment:
why do you need this?

Reply:
It is something like habbit.It seems bad one :) Removed.